blenderAddons/vispCAOExport/setup.py

```diff
@@ -0,0 +1,9 @@
+import bpy
+import sys
+import os
+
+print("BLENDER VERSION : " + bpy.app.version_string)
+ADDON_PATH = (bpy.utils.user_resource('SCRIPTS', "addons"))
+
+os.system("cp -r ../vispCAOExport " + ADDON_PATH)
+print("Moved addon to " + ADDON_PATH)
```

Comment:
You are right. `Install from file` will be certainly more intuitive for the user.
I propose that we keep the script for the moment and update the `README.md` with the instructions to import the zip created using `zip -r vispCAOExport.zip vispCAOExport`